src/BreadcrumbItem.js

```diff
@@ -39,7 +41,10 @@ class BreadcrumbItem extends React.Component {
     const linkProps = { href, title, target };
 
     return (
-      <li className={classNames(className, classes, { active })}>
+      <li
+        className={classNames(className, classes, { active })}
+        {...(active ? { 'aria-current': 'page' } : {})}
```

Comment:
I believe that props would get sent to the DOM if they are undefined so you could simplify this like:

```jsx
aria-current={active ? 'page' : undefined}
```